vuepress/docs/next/docs/getting-started/multitenancy.md

```diff
@@ -0,0 +1,46 @@
+---
+sidebarDepth: 2
+---
+
+# Entando Multitenancy
+
+## Overview
+
+Entando 7.2 introduces multitenancy to the Entando Platform. An Entando Application can be customized to enable a multitenant architecture where tenants share an infrastructure but are securely informationally isolated. This document provides an overview of multitenancy and its Entando implementation.
+
+## Core Concepts
+
+Software multitenacy describes an architecture in which a single software instance runs on a server and serves multiple tenants. Each tenant constitutes a group of users who share specific access privileges to the software instance. A multitenant software application is designed to provide a common architecture while ensuring the segregation of information.
+
+Entando Multitenancy imposes a shared architecture (e.g. the versions of Entando, K8s and Keycloak) while distributing resources (e.g. CPU, memory) across the primary and secondary tenants. The primary tenant is the user group with full access to the default features and functionalities of an Entando instance. Secondary tenants are user groups who share the privileges and capabilities of the primary tenant. 
+
+>Note: In the initial implementation of Entando Multitenancy, an active Component Manager is not available to secondary tenants.
```

Comment:
"an active Component Manager" -> "the Local Hub"

Reply:
would it be a good idea to include what this means, that only the primary has access to add and remove widgets and plugins available to the tenant<< something about this?